joyce-blair.css

```diff
@@ -0,0 +1,20 @@
+body {
+  font-family: myriad, sans-serif;
+  line-height: 1.4em;
+}
+h1 {
+  color: red;
+}
+#greenLine {
+  color: green;
+}
+.year {
+  font-weight: bold;
+  font-style: italic;
+}
+ul > ul > ul > li span {
```

Comment:
I could not figure that out at all!! I’m glad there wasn’t a gun to my head…J

 

 

From: Tom Kraak [mailto:notifications@github.com] 
Sent: Tuesday, March 01, 2016 4:25 PM
To: betamore/fewd-css-exercise-1
Cc: Joyce Blair
Subject: Re: [fewd-css-exercise-1] Css exercise1 (#8)

 

In joyce-blair.css <https://github.com/betamore/fewd-css-exercise-1/pull/8#discussion_r54638111> :

> @@ -0,0 +1,20 @@
> +body {
> +  font-family: myriad, sans-serif;
> +  line-height: 1.4em;
> +}
> +h1 {
> +  color: red;
> +}
> +#greenLine {
> +  color: green;
> +}
> +.year {
> +  font-weight: bold;
> +  font-style: italic;
> +}
> +ul > ul > ul > li span {

I love that child selector! :)

—
Reply to this email directly or view it on GitHub <https://github.com/betamore/fewd-css-exercise-1/pull/8/files#r54638111> .  <https://github.com/notifications/beacon/AQW9LncITj2C1JUnjQpgBict_Up8g2vmks5ppK6ugaJpZM4Hm3GP.gif>